java-manta-client-unshaded/src/test/java/com/joyent/manta/client/MantaClientConnectionFailuresIT.java

```diff
@@ -112,13 +112,20 @@ public NoHttpResponseMantaConnectionFactory(ConfigContext config, KeyPair keyPai
         }
     }
 
+    /**
+     * Test is disabled in testng.xml, the MantaClient constructor needs to be revisted.
+     *
+     * @throws IOException
+     */
+    @Test(enabled = false)
```

Comment:
Is the intention here to fix this resolve this before merging?

Reply:
No, this test has been disabled since [December 2016](https://github.com/joyent/java-manta/commit/49c0e5d5a1f993b5e86907a24b60ef61b42da875) and has been documented in #354 to be addressed soon (but not here).